### PR TITLE
Add secret.h

### PR DESCRIPTION
**Pull request by Edrig**
Add secret.h file to manage private variable

### CODE REVIEW

File: twitch_chat_ws2812b/twitch_chat_ws2812b.ino [hunk 1]
```diff
@@ -44,10 +46,10 @@ WebSocketsClient webSocket;
 StaticJsonDocument<200> doc;
 
 // Parameters for Twitch channel
-#define TWITCH_OAUTH_TOKEN "YOUR_OAUTH_TOKEN" // per Twitch's API 
-                                              // developer guide: https://twitchapps.com/tmi
-#define TWITCH_NICK "YOUR_CHANNELS_NAME"      // e.g., acrobotik
-#define TWITCH_CHANNEL "#YOUR_CHANNELS_NAME"  // e.g., #acrobotik
+const char twitch_oauth_token[] = "YOUR_OAUTH_TOKEN";
```
Comment (datasith) [inline twitch_chat_ws2812b/twitch_chat_ws2812b.ino]: shouldn't it be
```
const char twitch_oauth_token[] = TWITCH_OAUTH_TOKEN;
```
?

**Comment by Edrig** [comment]
@themakerbro its good now ^^

File: twitch_chat_ws2812b/twitch_chat_ws2812b.ino [hunk 2]
```diff
@@ -46,14 +46,14 @@ WebSocketsClient webSocket;
...
 
-const char twitch_nick[] = "TWITCH_NICK ";    
-const char twitch_channel[] = "TWITCH_CHANNEL";
+const char twitch_nick[] = TWITCH_NICK;    
+const char twitch_channel[] = TWITCH_CHANNEL;
 
 // Define necessary parameters for controlling the WS2812B LEDs
 #define N_PIXELS  16
-#define LED_PIN   19 // labeled "D6" on the board
+#define LED_PIN   0 // labeled "D6" on the board
```
Comment (datasith) [inline twitch_chat_ws2812b/twitch_chat_ws2812b.ino:56]: sorry, can we revert this back to 19? most examples seem to use pin D6 by default so it's in keeping with that :)

**Comment by datasith** [comment]
i'll fix the default pin for the next commit :)

**Pull request merged**